### DISCUSSION TITLE
linked DOIs not displayed when screening

### DISCUSSION THREAD

**Discussion by bmkramer** [asreview/asreview#630] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
I am not seeing linked DOIs when screening, while the data contain a column DOI. 
(as per the documentation: https://asreview.readthedocs.io/en/latest/intro/datasets.html#id4) 

Is there an easy fix for this? :bulb:

![image](https://user-images.githubusercontent.com/13157018/121687689-ee92db00-cac2-11eb-84ec-78f4af32a5e9.png)

**Comment by Rensvandeschoot** [comment, marked as answer]
@bmkramer thank you for reporting this issue! It seems the software does not recognize the digital object identifier with capitals. If you change `DOI` into `doi` it does work like it should. See also the documentation: https://asreview.readthedocs.io/en/latest/intro/datasets.html#id4
